extensions/ql-vscode/src/data-extensions-editor/external-api-usage.ts

```diff
@@ -25,10 +21,17 @@ export type ExternalApiUsage = {
    * The method parameters, including enclosing parentheses, e.g. `(String, String)`
    */
   methodParameters: string;
+}
+
+export interface ExternalApiUsage extends MethodSignature {
```

Comment:
This could be converted to `type ExternalApiUsage = MethodSignature & {...` if we want to keep that format. We seem to to not be using `interface` in the data extensions code.